cmd/tusd/cli/hooks.go

```diff
@@ -58,6 +58,14 @@ func SetupPreHooks(config *handler.Config) error {
 			MaxRetries: Flags.HttpHooksRetry,
 			Backoff:    Flags.HttpHooksBackoff,
 		}
+	} else if Flags.GrpcHooksEndpoint != "" {
+		stdout.Printf("Using '%s' as the endpoint for hooks", Flags.GrpcHooksEndpoint)
```

Comment:
Please make it in the log statement clear that we are talking about gRPC hooks, e.g. "using ... as the endpoint for gRPC hooks"